io.gpodder.gPodder.json

```diff
@@ -0,0 +1,77 @@
+{
+  "app-id": "io.gpodder.gPodder",
+  "runtime": "org.gnome.Platform",
+  "runtime-version": "3.26",
+  "sdk": "org.gnome.Sdk",
+  "command": "gpodder",
+  "rename-icon": "gpodder",
+  "rename-desktop-file": "gpodder.desktop",
+  "finish-args": [
+    "--socket=x11", "--share=ipc",
+    "--socket=system-bus",
```

Comment:
which interfaces do you need to talk to? this permission is too broad.

Reply:
Not sure. The accessibility interfaces at the very least, which I was initially getting errors for on launch. But gPodder needs to communicate with media players and devices, so I'm open to suggestions. New at this, this is my first flatpak, and I haven't done much GNOME app development before so I'm open to suggestions.

Reply:
OK. what you should do is run it with --log-system-bus on the command line and extract the interfaces it wants to talk to and then add them to finish args here. Here's an example of another app owning it's Mpris name on the bus, which is likely something you'll want: https://github.com/flathub/de.haeckerfelix.gradio/blob/3964c87227d35f1e4845a60332d5ceb0fc12af18/de.haeckerfelix.gradio.json#L19